crates/telio-lana/src/event_log_file.rs

```diff
@@ -187,11 +187,11 @@ pub mod moose {
     }
 
     /// Mocked moose function.
-    pub fn set_context_application_version(
+    pub fn set_context_application_libtelioapp_version(
```

Comment:
This change doesn't make sense imho - either application or libtelioapp, both sound redundant

Reply:
These function names should be the same as in `libmoose`. And in v3.0.0 they changed API a bit, so need to sync that here.